### PR TITLE
Add spaceradar, a disk visualization app

### PR DESCRIPTION
**Pull request by Amorymeltzer**
Modeled off a recent [electron-based submission by a particularly shady character](https://github.com/caskroom/homebrew-cask/pull/15665)

### CODE REVIEW

File: Casks/spaceradar.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask :v1 => 'spaceradar' do
+  version '4.4.0'
+  sha256 'de93d62bb2845fa815b7010a5f42960ce849487489370fae3636177944b083ff'
+
+  url "https://github.com/zz85/space-radar/releases/download/v#{version}/SpaceRadar-darwin-x64.zip"
```
Comment (vitorgalvao) [inline Casks/spaceradar.rb]: Please add a `appcast 'https://github.com/zz85/space-radar/releases.atom'` line under.

**Comment by vitorgalvao** [comment]
Your link is linking to itself.

I’m interested in knowing what you intended to link to, as I’m not aware of anything shady about this, and just recently directed a friend towards this app, since she has been trying disk visualisation apps. If there’s anything to be aware from it, I’d really like to know.

**Comment by Amorymeltzer** [comment]
Bah, slippery `C-w` finger.  The link should be fixed too (anchor fail).  ProTip: It's you

**Comment by vitorgalvao** [comment]
Ah, you mean the cask itself is modelled after another cask, not the app being modelled after another app, with shady origins. That’s a relief.

You’re also an Overcast user, then (I see you starred it)? Hope you enjoy it.

**Comment by vitorgalvao** [comment]
Merged as 9fc269f5a7a038e829d7f00e62c62bc5c1de4584.

**Comment by Amorymeltzer** [comment]
The wife is :wink:

**Comment by vitorgalvao** [comment]
Hope *she* enjoys it, then. Still the first version, but it’s mostly done for what I wanted from it, and have been using it for a while without major issues.

**Comment by zz85** [comment]
This is the first time I've heard of homebrew-cask. Thanks for adding the app!

**Pull request closed without merging**